src/command_modules/azure-cli-network/azure/cli/command_modules/network/custom.py

```diff
@@ -2043,7 +2043,7 @@ def create_lb_rule(
                                            backend_address_pool_name),
         probe=_get_property(lb.probes, probe_name) if probe_name else None,
         load_distribution=load_distribution,
-        enable_floating_ip=floating_ip == 'true',
+        enable_floating_ip=floating_ip,
```

Comment:
Should this [line](https://github.com/Azure/azure-cli/pull/8391/files#diff-0e8f50170282f3b6ea824007308bb5c1R2027) be changed?

Reply:
Yes. Thanks for the catch.